src/analysis/retail/priest/holy/modules/Abilities.ts

```diff
@@ -238,7 +230,7 @@ class Abilities extends CoreAbilities {
         healSpellIds: [SPELLS.HOLY_NOVA_HEAL.id],
       },
       {
-        spell: SPELLS.HOLY_WORD_CHASTISE.id,
+        spell: SPELLS.HOLY_WORD_CHASTISE_TALENT.id,
```

Comment:
```suggestion
        spell: TALENTS.HOLY_WORD_CHASTISE_TALENT.id,
```

Reply:
Did this for every spell made into a talent for Dragonflight and replaced dependencies in all files